static/elements/chromedash-form-field.js

```diff
@@ -68,14 +62,10 @@ export class ChromedashFormField extends LitElement {
     return this;
   }
 
-  render() {
-    const label = this.fieldProps.label;
-    const helpText = this.fieldProps.help_text || '';
-    const extraHelpText = this.fieldProps.extra_help || '';
+  renderField() {
```

Comment:
Since the whole thing is the field, maybe this should be named `renderWidgets()`.

Reply:
Changed!